tools/ocb-sync.sh

```diff
@@ -8,7 +8,7 @@
 
 ODOO="git@github.com:/odoo/odoo.git"
 OCB="git@github.com:/OCA/OCB.git"
-BRANCHES="18.0 17.0 16.0 15.0"
+BRANCHES="19.0 18.0 17.0 16.0 15.0"
```

Comment:
Let's remove 15? it's very unlikely to evolve by now.

Reply:
Well, Odoo will start maintaining 5 versions, isn't it?